apps/signing/tests/test_views.py

```diff
@@ -104,6 +104,16 @@ def test_version_already_exists(self):
         assert response.status_code == 409
         assert response.data['error'] == 'Version already exists.'
 
+    @mock.patch('devhub.views.Version.from_upload')
+    def test_no_version_yet(self, from_upload):
+        response = self.put(self.url(self.guid, '3.0'))
+        assert response.status_code == 202
+        assert 'processed' in response.data
+
+        response = self.get(self.url(self.guid, '3.0'))
```

Comment:
it's not totally obvious from the test name why this is here. Ideally, I like to see things such as this in a separate test function but at the least I think you could just add a comment to explain what it's testing for.